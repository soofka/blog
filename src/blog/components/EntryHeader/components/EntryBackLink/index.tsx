import * as React from 'react';
import { Link } from 'react-router-dom';

import RoutingProvider from 'common/RoutingProvider';

import Label from 'components/Label';

export const EntryBackLink = (): JSX.Element => (
  <Link to={RoutingProvider.getHomeNiceUrlBasePath()}>
    &laquo; {<Label name="entry__back" />}
  </Link>
);

export default EntryBackLink;
