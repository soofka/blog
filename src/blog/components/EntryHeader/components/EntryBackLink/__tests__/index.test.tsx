import * as React from 'react';
import { shallow } from 'enzyme';

import EntryBackLink from 'components/EntryHeader/components/EntryBackLink';

describe('<EntryBackLink/>', () => {

  it('renders correctly', () => {
    expect(shallow(
      <EntryBackLink
        created="testCreated"
        updated="testUpdated"
      />,
    )).toMatchSnapshot();
  });

});
