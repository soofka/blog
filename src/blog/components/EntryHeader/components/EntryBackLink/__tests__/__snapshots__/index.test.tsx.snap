// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EntryBackLink/> renders correctly 1`] = `
<Link
  replace={false}
  to="/"
>
  Â«
  <inject-_class-with-languageStore
    name="entry__back"
  />
</Link>
`;
